### ISSUE TITLE
Totally missing any notification when game starts

### ISSUE THREAD

**Issue by Licho1** [ZeroK-RTS/Chobby#198]
1) join some PW defense
2) alt tab to do other stuff
3) wait 10 minutes
4) your game starts you wont notice anything

**Comment by gajop** [comment]
Yep, I noticed the same thing, but what should we do? I don't know the PW design, but how come there isn't a ready check? What would happen if the user is away/failed the ready check?

Adding the notification alone wouldn't be enough if the wait time is long.

**Comment by Licho1** [comment]
There should be general notification upon ConnectSpring command. it should consist of window flash (if inactive) and a sound. 
It can be optional only if you are not in active lobby.
